services/madoc-ts/src/frontend/site/pages/search.tsx

```diff
@@ -1,146 +1,66 @@
-                          <LocaleString>{item.label}</LocaleString>
-                        </SearchFilterLabel>
-                        {showSearchFacetCount ? (
-                          <SearchFilterItemCount>
-                            {t('{{count}} manifests', { count: item.count })}
-                          </SearchFilterItemCount>
-                        ) : null}
-                        {isSelected !== 0 ? (
-                          <SearchFilterToggle onClick={() => clearSingleFacet(item.key, item.values)}>
-                            <CloseIcon $solid width={12} height={12} />
-                          </SearchFilterToggle>
-                        ) : (
-                          <SearchFilterToggle onClick={() => applyFacet(item.key, item.values)}>
-                            <AddIcon height={12} width={12} />
-                          </SearchFilterToggle>
-                        )}
-                      </SearchFilterItem>
-                    );
-                  })}
-                </SearchFilterItemList>
-              </SearchFilterSection>
-            );
-          })}
-        </SearchFilterContainer>
-        <div style={{ flex: '1 1 0px' }}>
-          {isLoading && !searchResponse ? (
-            <LoadingBlock />
-          ) : (
-            <TotalResults>
-              {t('Found {{count}} results', {
-                count: searchResponse && searchResponse.pagination ? searchResponse.pagination.totalResults : 0,
-              })}
-            </TotalResults>
-          )}
-          <Pagination
-            page={page}
-            totalPages={latestData && latestData.pagination ? latestData.pagination.totalPages : undefined}
-            stale={isLoading}
-            extraQuery={rawQuery}
-          />
-          <SearchResults
-            isFetching={isLoading}
-            value={fulltext}
-            searchResults={searchResponse ? searchResponse.results : []}
-          />
-          <Pagination
-            page={page}
-            totalPages={latestData && latestData.pagination ? latestData.pagination.totalPages : undefined}
-            stale={isLoading}
-            extraQuery={rawQuery}
-          />
+        <div style={{ maxWidth: 300 }}>
+          <Slot name="search-filters1" small>
+            <SearchPageFilters />
+          </Slot>
+        </div>
+
+        <div style={{ width: '100%' }}>
+          <Slot name="search-page-pagination">
+            <SearchPagination />
+          </Slot>
+
+          <Slot name="search-page-resultse">
+            <AppliedFacets />
+            {isLoading && !searchResponse ? (
+              <LoadingBlock />
+            ) : (
+              <TotalResults>
+                {t('Found {{count}} results', {
+                  count: searchResponse && searchResponse.pagination ? searchResponse.pagination.totalResults : 0,
+                })}
+              </TotalResults>
+            )}
```

Comment:
I think this would need to be a page block too, otherwise it will disappear after customising it